sunbeam-python/sunbeam/commands/prepare_node.py

```diff
@@ -45,10 +45,13 @@
     rm -f /tmp/90-$USER-sudo-access
 }}
 
-# Ensure OpenSSH server is installed
-dpkg -s openssh-server &> /dev/null || {{
-    sudo apt install -y openssh-server
-}}
+# Ensure pre-reqs - e.g. OpenSSH server are installed
+PREREQ_DPKGS=("curl" "openssh-server")
+for pkg in ${PREREQ_DPKS[@]}; do
```

Comment:
Should be PREREQ_DPKGS

Reply:
Agree, fixed.

Reply:
Sorry about that one

Reply:
tox checks seem to dislike the bash formatting for arrays, but since this is heredoc bash and not py thats to be expected i guess.

Reply:
Reverted to the simpler approach. Was a nice idea but the templating and escaping caused to many issues for adding one extra package.